ruby_event_store/lib/ruby_event_store/async_dispatcher.rb

```diff
@@ -0,0 +1,20 @@
+module RubyEventStore
+  class AsyncDispatcher < PubSub::Dispatcher
+    def initialize(proxy_strategy: AsyncProxyStrategy::Inline.new, scheduler:)
```

Comment:
For a bried moment I've considered passing some default `scheduler` that raises on `call` and `async_handler?` but required keyowrd seems fair enough.